server/historian/package.json

```diff
@@ -15,7 +15,9 @@
     "build:compile": "lerna run build:compile --stream",
     "build:genver": "lerna run build:genver --stream --parallel",
     "clean": "rimraf dist *.tsbuildinfo *.build.log",
+    "format": "lerna run format --no-sort --stream -- -- -- --color",
     "postinstall": "npm run postinstall:lerna",
+    "lerna": "lerna",
```

Comment:
This shouldn't be needed. Was something not working without this?

Reply:
Looks like this was added to a couple other package.json files. We'll want to remove them there too.